src/Components/Artwork/FlatGridItem.tsx

```diff
@@ -88,6 +89,7 @@ const FlatGridItem: React.FC<FlatGridItemProps> = ({ artwork, onClick }) => {
               width="100%"
               height="100%"
               style={{ display: "block" }}
+              placeHolderURL={blurHashDataURL ?? undefined}
```

Comment:
To simplify this, we should just accept null on this prop in Palette.

Reply:
True - it's typed here - https://github.com/artsy/palette/blob/60f91aef456c0ab2aafad1ad71e52053171aa9fb/packages/palette/src/elements/Image/Image.tsx#L32 - optional so `undefined` works, but not `null`.